src/apps/snmp/snmp_traps.c

```diff
@@ -343,6 +343,15 @@ snmp_send_msg(struct snmp_msg_trap *trap_msg, struct snmp_varbind *varbinds, u16
  * and .iso.org.dod.internet.private.enterprises.yourenterprise
  * (sysObjectID) for specific traps.
  */
+
+// *** SPAN ***
+// Disable dangling pointer for this function until it's improved upstream.
+// Does not compile on armgcc 13.2 otherwise.
+#if defined(__GNUC__) || defined(__GNUG__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdangling-pointer"
```

Comment:
I think this line will only work _for_ newer versions of gcc - earlier versions will error out about an unknown option or something

Reply:
:+1: added a `#if __GNUC__ >= 13` check